language/use-cases/sensitive-data-detection/context_aware_sensitive_data_detection.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            if value is not None:
</pre><p><code>if value:</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>